reference/5.1/Microsoft.PowerShell.Core/About/about_numeric_literals.md

```diff
@@ -0,0 +1,190 @@
+---
+ms.date:  04/09/2018
+schema:  2.0.0
+locale:  en-us
+keywords:  powershell,cmdlet
+title:  About numeric literals
+---
+# About numeric literals
+
+There are two kinds of numeric literals: integer and real. Both can have type
+and multiplier suffixes.
+
+## Integer literals
+
+Integer literals can be written in decimal or hexadecimal notation. Hexadecimal
+literals are prefixed with "0x" to distinguish them from decimal numbers.
+
+Integer literals can have a type suffix and a multiplier suffix.
+
+| Suffix |       Meaning       |
+| ------ | ------------------- |
+| l      | long data type      |
+| kb     | kilobyte multiplier |
+| mb     | megabyte multiplier |
+| gb     | gigabyte multiplier |
+| tb     | terabyte multiplier |
+| pb     | petabyte multiplier |
+
+The type of an integer literal is determined by its value, the type suffix, and
+the numeric multiplier suffix.
+
+For an integer literal with no type suffix:
+
+- If the value can be represented by type `[int]`, that is its type.
+- Otherwise, if the value can be represented by type `[long]`, that is its
+  type.
+- Otherwise, if the value can be represented by type `[decimal]`, that is its
+  type.
+- Otherwise, it is represented by type `[double]`.
+
+For an integer literal with a type suffix:
+
+- If the type suffix is `u` and the value can be represented by type `[int]`
+  then its type is `[int]`.
+- If the type suffix is `u` and the value can be represented by type `[long]`
+  then its type is `[long]`.
+- If its value can be represented by type specified then that is its type.
+- Otherwise, that literal is ill formed.
```

Comment:
ill-formed should always be hyphenated.  Fix on other instances, too.

Reply:
Changed to `malformed` - a better choice